00_DO_FIRST.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
<p>No google drive, we should use purely public data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/rapids_singlecell-notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
E.g. https://figshare.com/articles/dataset/93k_Human_Lung_Tumor_data/25664775?file=45788454